sensing/autoware_pointcloud_preprocessor/src/blockage_diag/blockage_diag_node.cpp

```diff
@@ -190,6 +190,12 @@ void BlockageDiagComponent::filter(
     sky_blockage_range_deg_[1] = angle_range_deg_[1];
   } else {
     for (const auto p : pcl_input->points) {
+      if (p.channel > vertical_bins) {
```

Comment:
@vividf Thank you for your PR, I have a minor suggestion since p.channel valua range is [0, vertical_bins -1].
```suggestion
      if (p.channel >= vertical_bins) {
```

Reply:
@badai-nguyen thanks for the check!